src/blocks/blocks/countdown/style.scss

```diff
@@ -2,30 +2,38 @@
 	--background-color: transparent;
 	--border-color: #000;
 	--border-radius: 0px;
+
 	--width: 100%;
 	--container-width: clamp(400px, var(--width) * 4, 100%);
 	--container-width-tablet: var(--container-width);
 	--container-width-mobile: var(--container-width-tablet);
+
 	--height: 100px;
-	--height-tablet: 100px;
-	--height-tablet: var(--height-tablet);
+	--height-tablet: var(--height);
```

Comment:
You sure it's supposed to be `--height` and not `--height-tablet`?

Reply:
--height-tablet will inherit from --height